lib/galaxy/workflow/modules.py

```diff
@@ -992,7 +992,7 @@ def callback(input, prefixed_name, context, **kwargs):
                 options = None
                 if static_options and len(static_options) == 1:
                     # If we are connected to a single option, just use it as is so order is preserved cleanly and such.
-                    options = [{"value": o[0], "label": o[1]} for o in static_options[0]]
+                    options = [{"value": o[1], "label": o[0]} for o in static_options[0]]
```

Comment:
```suggestion
                    options = [{"label": o[0], "value": o[1]} for o in static_options[0]]
```

Reply:
(just for my sanity and consistency in order of indices...)

Reply:
sounds good, thanks